src/js/pages/catalog/PackageDetailTab.js

```diff
@@ -648,4 +685,4 @@ const renderUnresolvedDependency = dependency => (
   </div>
 );
 
-module.exports = PackageDetailTab;
+module.exports = withI18n({ withRef: true })(PackageDetailTab);
```

Comment:
We need {withRef: true} in order to have the unit test working.
https://github.com/dcos/dcos-ui/pull/4076/files#diff-2f2fdab2ce3beac9be3e39f63da1fe81R25

Reply:
We can get the `i18n` object from `container` instead of using `withI18n` and mock it in unit tests. I've done this in a few places in the code to avoid breaking existing unit tests when adding `withI18n`

Reply:
i don't understand why we use that `props.i18n._`-stuff or Inversify here.

if we use `Trans` the text is extractable and the tests run just fine... (after removing `getWrappedInstance` again)